install_config/aggregate_logging.adoc

```diff
@@ -3628,3 +3636,304 @@ cluster. Edit your non-cluster logging service (for example, `logging-es`,
 ----
 $ oc patch svc/logging-es -p '{"spec":{"selector":{"component":"es","provider":"openshift"}}}'
 ----
+
+[[optimal-state-cluster]]
+=== Cluster not in an optimal state
```

Comment:
We're moving towards sentence case, but aren't quite there yet. To maintain consistency within the topic: 

s/Cluster not in an optimal state/Cluster Not in an Optimal State